packages/typography/src/Description.tsx

```diff
@@ -20,7 +20,7 @@ const descriptionTypeScale: Record<BaseFontSize, string> = {
   `,
   [BaseFontSize.Body2]: css`
     font-size: ${typeScales.body2.fontSize}px;
-    line-height: ${typeScales.body2.lineHeight}px;
+    line-height: 20px;
```

Comment:
Why was this reverted to `20px`? It looks like `typeScales.body1.lineHeight`would work here